tests/server/test_env.py

```diff
@@ -406,3 +406,16 @@ def test_ban_versions():
 def test_coin_class_provided():
     e = Env(lib_coins.BitcoinSV)
     assert e.coin == lib_coins.BitcoinSV
+
```

Comment:
Should add one more space for PEP 8-compliance

Reply:
Since I was there, I ran the whole file thru autopep8